src/bidiMapper/CommandProcessor.ts

```diff
@@ -42,9 +42,9 @@ export interface BidiParser {
   parseDisownParams(params: object): Script.DisownParameters;
   parseSendCommandParams(params: object): CDP.SendCommandParams;
   parseGetSessionParams(params: object): CDP.GetSessionParams;
+  parseSubscribeParams(params: object): Session.SubscribeParameters;
```

Comment:
I wonder what this re-arrange improves

Reply:
It helps when adding new methods. All `BrowsingContext` method signatures are now grouped together.

Previously, `Session` was in the middle of `BrowsingContext`.